docs/source/conf.py

```diff
@@ -89,6 +89,18 @@
 # The full version, including alpha/beta/rc tags.
 release = cartopy.__version__
 
+# Sphinx gallery configuration
+sphinx_gallery_conf = {
+    'examples_dirs': ['../../lib/cartopy/examples'],
+    'filename_pattern': '^((?!sgskip).)*$',
+    'gallery_dirs': ['gallery'],
+    'doc_module': ('cartopy'),
+    'reference_url': {'matplotlib': 'http://matplotlib.org',
+                      'numpy': 'http://docs.scipy.org/doc/numpy/reference',
+                      'scipy': 'http://docs.scipy.org/doc/scipy/reference'},
```

Comment:
Might be a problem due to sphinx-gallery/sphinx-gallery#236, but it should be fixed with sphinx-gallery/sphinx-gallery#258.